source/overloading.tex

```diff
@@ -1604,7 +1604,7 @@
 \pnum
 When resolving a placeholder for a deduced class type\iref{dcl.type.simple}
 where the \grammarterm{template-name} names an alias template \tcode{A},
-the \grammarterm{defining-type-id} of \tcode{A} must be of the form
+the \grammarterm{defining-type-id} of \tcode{A} shall be of the form
```

Comment:
We have an "as specified in" below pointing to a normative rule elsewhere, so I think "must" here was OK following our "internal constraint" meaning.
```suggestion
the \grammarterm{defining-type-id} of \tcode{A} must be of the form
```

Reply:
done